src/filters/Resize.ts

```diff
@@ -1,12 +1,26 @@
 // @ts-nocheck
-
 import type { TClassProperties } from '../typedefs';
 import { BaseFilter } from './BaseFilter';
 import type { T2DPipelineState, TWebGLPipelineState } from './typedefs';
 import { isWebGLPipelineState } from './typedefs';
 import { classRegistry } from '../ClassRegistry';
 import { createCanvasElement } from '../util/misc/dom';
 
+export const resizeDefaultValues: Partial<TClassProperties<Resize>> = {
+  resizeType: 'hermite',
+  scaleX: 1,
+  scaleY: 1,
+  lanczosLobes: 3,
+  fragmentSourceTOP: `
+    precision highp float;
+    uniform sampler2D uTexture;
+    uniform vec2 uDelta;
+    varying vec2 vTexCoord;
+  `,
```

Comment:
for consistency should be moved to a standalone file?

Reply:
this resize one is too messy. but shouldn't be a property either, i will move this